src/components/Board.jsx

```diff
@@ -0,0 +1,77 @@
+'use client';
+
+import Image from 'next/image';
+import { useState, useEffect } from 'react';
+
+function UserList() {
+    const [users, setUsers] = useState([]);
+
+    useEffect(() => {
+        fetch('https://dummyjson.com/users?limit=15')
+            .then(res => res.json())
+            .then(data => {
+                if (data && Array.isArray(data.users)) {
+                    setUsers(data.users);
+                } else {
+                    console.error('Error: Invalid or missing user data');
+                }
+            })
+            .catch(error => console.error("Error fetching users:", error))
+    }, []);
+
+    const sortedUsers = users.sort((a, b) => b.weight - a.weight);
+
+    return (
+        <div className="mt-4">
+             {sortedUsers.length > 0 ? (
+                users.map((user, index) => {
+                    return (
+                        <div className="flex-between" key={index}>
+                            <div className="flex-between">
+                                <Image src={user.image} alt="user.username" width={40} height={40} />
+                                <h3 className="p-2">
+                                    {user.firstName} {user.lastName}
+                                </h3>
+                            </div>
+                            <div className="flex-between">
```

Comment:
There are no hints on the top of the table columns - points and hours. It will be unclear to the user.